orchestrator_service/src/main/java/com/override/orchestrator_service/service/TransactionProcessingService.java

```diff
@@ -31,21 +29,69 @@ public class TransactionProcessingService {
     @Autowired
     private ExecutorService executorService;
 
-    private final String SPACE = " ";
+    private enum AmountPositionType {
+        AMOUNT_IN_FRONT,
```

Comment:
Возможно темка спорная, мне кажется, что луше б назвать "начало и конец"
```suggestion
        AMOUNT_AT_BEGINNING, 
        AMOUNT_AT_END,
        AMOUNT_AT_BEGINNING_RU_LOCALE,
        AMOUNT_AT_END_RU_LOCALE,
```  
или  

```suggestion
        BEGINNING_AMMOUNT, 
        END_AMMOUNT,
        BEGINNING_AMMOUNT_RU_LOCALE,
        END_AMMOUNT_RU_LOCALE,
```

Reply:
я не англичанин, но посчитал, что так будет понятно
нужны еще мнения

Reply:
я за первый вариант Бориса

Reply:
переименовал как в первом варианте предложил Борис